GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -104,13 +104,16 @@
     private GVRScreenshotCallback mScreenshotCenterCallback = null;
     private GVRScreenshotCallback mScreenshotLeftCallback = null;
     private GVRScreenshotCallback mScreenshotRightCallback = null;
+    private GVRScreenshot3DCallback mScreenshot3DCallback = null;
+    ByteBuffer readbackBuffer = null;
```

Comment:
We all know that I am no fan of the `'m'` prefix ... but consistency is important. If you add a field right next to other fields that have an `'m'`, it too should have an `'m'`.